### ISSUE TITLE
authorization agent should check Resource Owner's address book if it includes requesting End-user's WebID

### ISSUE THREAD

**Issue by elf-pavlik** [solid/data-interoperability-panel#96]
Based on conversation in https://github.com/solid/authentication-panel/issues/161#issuecomment-817069747

TL;DR phishing WebIDs using different cases IRIs or characters that just look similar to human eye

As we work on workflows to request access, in cases where [End-user](https://www.ietf.org/archive/id/draft-ietf-gnap-core-protocol-04.html#section-1.2-2.10.1) (not a [Client](https://www.ietf.org/archive/id/draft-ietf-gnap-core-protocol-04.html#section-1.2-2.4.1)) requests access, authorization agent should check if that End-user exists in [Resource Owner](https://www.ietf.org/archive/id/draft-ietf-gnap-core-protocol-04.html#section-1.2-2.4.1)'s address book and mark it clearly on the consent screen.

**Comment by elf-pavlik** [comment]
related #214

**Comment by elf-pavlik** [comment]
Overall work is being tracked in #302, a planned approach that verification and creates agent registrations happens before any specific data is shared or access requested.
